src/lib/eap/base.c

```diff
@@ -115,39 +115,33 @@ fr_dict_attr_autoload_t eap_base_dict_attr[] = {
 	{ NULL }
 };
 
-fr_pair_t *eap_packet_to_vp(fr_radius_packet_t *packet, eap_packet_raw_t const *eap)
+void eap_packet_to_vp(fr_radius_packet_t *packet, eap_packet_raw_t const *eap, fr_pair_list_t *list)
```

Comment:
https://wiki.freeradius.org/contributing/coding-standards#argument-order

Should be `int eap_packet_to_vp(TALLOC_CTX *ctx, fr_pair_list_t *list,  eap_packet_raw_t const *eap)`

Ctx should be `request->request_ctx` this must have been one of the ones I missed.